src/wp-content/themes/twentytwelve/js/navigation.js

```diff
@@ -3,6 +3,40 @@
  * accessibility for submenu items.
  */
 ( function() {
+	function getParentElements( element, selector ) {
```

Comment:
are these useful enough to be moved into a package for re-use? thinking if we are encouraging the community to transition away from jQuery, covering the top use cases in packages (either distinct - or perhaps bundled into a single `wp-dom` package for efficiency - would make sense vs repeating the same boilerplate code in each theme.

Reply:
I agree, it would be great to have a core-provided script with useful utility functions.

Most of what jQuery offers is nowadays not necessary, but it still has a few nice functions which replicating all the time is painful. Yet, loading jQuery for those is overkill. @mtias Is a simple DOM utility functions package (e.g. `wp-dom`) something we could consider moving forward through Gutenberg?